lib/run/index.js

```diff
@@ -376,6 +377,11 @@ module.exports = function (options, callback) {
                         console.warn('  please install reporter using npm\n');
                     }
                 }
+                // warn users if the reporter is not loaded
+                else if (!(_.isFunction(Reporter) && _.isObject(emitter.reporters[reporterName]))) {
```

Comment:
This will be redundant if we simply add another line in before console.warn in line 364 indicating there’s a problem with the reporter.